.github/workflows/unit-and-integration-tests.yaml

```diff
@@ -23,18 +23,48 @@ jobs:
     needs:
       - call-inclusive-naming-check
 
-  integration-tests:
-    name: Integration tests on Testflinger instance with Intel GPU
-    runs-on: [self-hosted, testflinger]
+  integration-tests-system-A:
```

Comment:
Would it be better to run a sequence of jobs for better scalability ? Here we are limited by the number of runners that is controlled by IS and Cert team

Reply:
The reason I used two sections is that it appeared there were two self-hosted runners deployed to the repo. It turns out there was only one but the cert team has since deployed a second runner for the repo, so I think it makes sense to keep two sections to allow the jobs to run in parallel. 

The integration tests take a long time to run (30 mins - 1 hour), so it's nice to get test results back about twice as fast.

Reply:
i understand why you want 2 jobs in parallel but as i mentioned; it limits the scalability, how will we do if we want to run the tests on a third platform ?

Reply:
and i do believe that if the tests take that long, we should consider to not do it on merge request but only daily on the main branch

Reply:
> Additional jobs would need to be added in sequence like you suggest, to either of the two sections. But sense we have two runners we might as well use them both.

Ok; we can do that

Reply:
> and i do believe that if the tests take that long, we should consider to not do it on merge request but only daily on the main branch

What would that be testing? The purpose of the integration tests is to ensure that code changes are not breaking the operator deployment in some way. 

One aspect that could be interesting in a daily (or maybe weekly would be enough) test is installing the latest version of dependencies (microk8s, juju, ops) rather than pinning to a specific version. That would test that nothing breaks when, for example, a new version of juju is released.

Reply:
ok that is ok for me